src/test-and-demo-constants.js

```diff
@@ -32,7 +32,6 @@ export const slideArray = [
 	{
 		slides: slides,
 		carouselOptions: {
-			navArrows: true,
```

Comment:
this should be updated to reflect new way of defining `navArrows`. along with most of the other ones in the file, except for where `navArrows` is set to `false`

Reply:
Not having navArrows defined is the same thing as setting to false. That's why just deleted it.

Reply:
Never mind, I'm an idiot, I did change the way arrows were feeding into the demo, but I forgot to push it from my laptop. Will do when I get home today.

Reply:
I made the commit but it isn't showing up on this pull request. I'll close it and reopen.